releases/R2023b/README.md

```diff
@@ -44,7 +44,7 @@ To deploy MATLAB Production Server onto an existing VPC, specify these additiona
 |--------------- | ----- |
 ||**Network License Manager**|
 | **Deploy License Server** | Specify whether you want to deploy the Network License Manager for MATLAB. This parameter is available only if you use the deployment template for an existing VPC. <p>If you are using an existing VPC, see the [Use Existing License Server in Existing VPC](#use-existing-license-server-in-existing-vpc) section.</p> |
```

Comment:
I think this sentence "If you are using an existing VPC, see the [Use Existing License Server in Existing VPC section]" does not feel right anymore. 
With the addition of the new optional parameters, "IP Address and port number of Existing Network License Manager" and "Security Group of Existing Network License Manager", we are automating what users had to manually configure earlier. These manual configuration steps are mentioned in the "Use Existing License Server in Existing VPC section". 
Users can choose not to enter anything in the two new optional parameters and configure it manually. 
Can we add some text to explain this?